addons/istio/1.6.x/istio-3.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: istio
+  labels:
+    kubeaddons.mesosphere.io/name: istio
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.6.4-2"
```

Comment:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.6.4-3"
```

Reply:
There's a JIRA to automate the revisions, but until that gets priority we have to manually manage them or we'll get collisions. :'(